tests/x11regressions/firefox/sle12/firefox_pagesaving.pm

```diff
@@ -40,8 +40,8 @@ sub run() {
     send_key "ctrl-l";
     wait_still_screen 3;
     type_string "ls Downloads/\n";
-    assert_screen('firefox-pagesaving-downloads', 10);
-    type_string "rm -rf Downloads/*\n";
+    assert_screen 'firefox-pagesaving-downloads';
+    script_run 'rm -rf Downloads/*';
```

Comment:
but then it should be `assert_script_run`. If this fails something serious went wrong and we should care about it. It might not be a problem of firefox but our system testing approach anyway mainly shows problems which are not related to the test modules name, right? ;-)